app/src/main/java/org/oppia/android/app/testing/FractionInputInteractionViewTestActivity.kt

```diff
@@ -0,0 +1,44 @@
+package org.oppia.android.app.testing
+
+import android.view.View
+import org.oppia.android.app.activity.InjectableAppCompatActivity
+import org.oppia.android.app.model.Interaction
+import org.oppia.android.app.model.WrittenTranslationContext
+import org.oppia.android.app.player.state.answerhandling.AnswerErrorCategory
+import org.oppia.android.app.player.state.answerhandling.InteractionAnswerErrorOrAvailabilityCheckReceiver
+import org.oppia.android.app.player.state.itemviewmodel.FractionInteractionViewModel
+import org.oppia.android.app.player.state.listener.StateKeyboardButtonListener
+import org.oppia.android.app.translation.AppLanguageResourceHandler
+import org.oppia.android.domain.translation.TranslationController
+import javax.inject.Inject
+
+class FractionInputInteractionViewTestActivity :
+  InjectableAppCompatActivity(),
+  StateKeyboardButtonListener,
+  InteractionAnswerErrorOrAvailabilityCheckReceiver {
+
+  @Inject
+  lateinit var resourceHandler: AppLanguageResourceHandler
+
+  @Inject
+  lateinit var translationController: TranslationController
+
+  val fractionInteractionViewModel by lazy {
```

Comment:
This class is missing `onCreate` function.
Take a look at the similar function in the `InputInteractionViewTestActivity`.  Add the UI logic too. 

Also, see `activity_input_interaction_view_test`, how we are binding the UI with the `InputInteractionViewTestActivity`. Now, for this PR, we need to separate out the fraction part completely in the new UI file as well.

Reply:
done